src/h5p-editor-csv-to-text.ts

```diff
@@ -82,28 +85,39 @@ class CSVToTextWidget extends H5PWidget<H5PFieldText> implements IH5PWidget {
 
   private static createFileInput(
     eventListener: (event: Event) => void,
-  ): HTMLInputElement {
+  ): HTMLDivElement {
     const inputId = H5P.createUUID();
+    
+    const fileInputWrapper = document.createElement("div");
+    fileInputWrapper.classList.add("file");
+    fileInputWrapper.classList.add("csvtotext-file");
+
+    const fileInputLabelText = document.createElement("div");
+    fileInputLabelText.classList.add("h5peditor-field-file-upload-text");
+    fileInputLabelText.innerHTML += "Import CSV-file";
```

Comment:
This label should probably be internationalized.

```suggestion
    // TODO: Translate
    fileInputLabelText.innerHTML += "Import CSV-file";
```